aspnetcore/performance/response-compression/sample/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace ResponseCompressionSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = new WebHostBuilder()

```

Comment:
@JunTaoLuo Doesn't `Sdk.Web` completely provide the `processPath` and `arguments`. My testing says 'yes,' but I'm not sure in the net451 case. If the `Sdk.Web` will add them both, then it seems like it would clean up sample files quite a bit.

The current `web.config` for these samples might go in the direction of IISIntegration sample ...
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <!-- Configure your application settings in appsettings.json. Learn more at http://go.microsoft.com/fwlink/?LinkId=786380 -->

  <system.webServer>
    <handlers>
      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
    </handlers>
    <!-- This set of attributes are used for launching the sample using IISExpress via Visual Studio tooling -->
    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
    
    <!-- This set of attributes are used for launching the sample for full CLR (net451) without Visual Studio tooling
    <aspNetCore processPath=".\IISSample.exe" arguments="" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
    -->
    
    <!-- This set of attributes are used for launching the sample for Core CLR (netcoreapp1.1) without Visual Studio tooling
    <aspNetCore processPath="dotnet" arguments=".\IISSample.dll" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
    -->
  </system.webServer>
</configuration>
```
but that could be (for the `Sdk.Web` world) ...

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <!-- Configure your application settings in appsettings.json. Learn more at http://go.microsoft.com/fwlink/?LinkId=786380 -->

  <system.webServer>
    <handlers>
      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
    </handlers>
    <aspNetCore stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
  </system.webServer>
</configuration>
```